packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -842,6 +842,8 @@ Adds a SAML 2.0 application. This application is only available to the org that
 
 | Parameter             | Description                                     | DataType                             | Nullable | Unique | Validation                              |
 | --------------------- | ----------------------------------------------- | ------------------------------------ | -------- | -----  | --------------------------------------  |
+| allowMultipleAcsEndpoints       | Determines whether the app allows configuring multiple ACS URIs                                         | Boolean                                              | FALSE    | FALSE  |                                           |
```

Comment:
Determines whether the app allows you to configure multiple ACS URIs

Reply:
done